### PR TITLE
Made the English translations a little more comprehensive

### PR DESCRIPTION
**Pull request by robcalcroft**
A few of the translations had some broken English in, I've cleared those up and re-written a couple to make them easier to read and understand :smile:. Please let me know if anything I have changed has altered the meaning.

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -89,11 +89,11 @@
...
-	"passenger-request-price.set_price": "In this app passenger sets the price 😉 Give us your price (in local currency). How much are you willing to pay? (or type \"0\" (zero) if you don't know)",
+	"passenger-request-price.set_price": "In this app the passenger sets the price 😉 Give us your price in your local currency. How much are you willing to pay? (or type \"0\" (zero) if you don't know)",
 	"passenger-request-price.set_price_desc": "Here we ask passenger to set the price for the order. In our app passengers set the price for orders.",
 	"passenger-request-price.should_be_numeric": "Oops, price should be the numeric value (and in local currency)! For example: \"50\". Please try again!",
 	"passenger-request-price.should_be_numeric_desc": "We show this error text to passenger when s/he entered the price for the order, but the value is not numeric.",
-	"passenger-request-price.on_timeout": "Seems like you're waiting for too long? Sorry about that. If you haven't found a ride, we recommend to try again later.",
+	"passenger-request-price.on_timeout": "Seems like you're been waiting for a while? Sorry about that. If you haven't found a ride, we recommend trying again later.",
```
Comment (jytug) [inline locales/en.json]: "You've been waiting for a while"

File: locales/en.json [hunk 2]
```diff
@@ -109,15 +109,15 @@
 	"passenger-verify-cash.verify_cash_desc": "We display this message to passengers before they can place order. If they don't have cash, on the next step we'll ask them to get some cash.",
 	"passenger-verify-cash.get_some_cash": "🙁 Sorry, you must have cash before you can have a ride. Please try to get some cash and try again!",
 	"passenger-verify-cash.get_some_cash_desc": "Here we ask users to try the different way of sending location. Instead of clicking the button, they can try to use Telegram menu.",
-	"driver-request-radius.request_radius": "At what distance from the passenger you would be interested in order? Select from the list 👇 or type the number (up to 10 km).",
+	"driver-request-radius.request_radius": "How far away from your current location do you want to travel to pickup a passenger? Select from the list 👇 or type the number (up to 10 km).",
```
Comment (jytug) [inline locales/en.json]: Either "pick up a passenger" or "pick a passenger up"

**Comment by ro31337** [comment]
Nice, @robcalcroft thanks for your contribution! May I ask you to fix few minor things @jytug found? Thanks

**Comment by Dannymatos73** [comment]
ten miles 50.00 would be the price

**Comment by ro31337** [comment]
@jytug thanks, improved here: https://github.com/ro31337/libretaxi/pull/472/commits/0dee347021271d1cc6c23dacb4623b33d55c4c6c

**Comment by robcalcroft** [comment]
Hey! Apologies I didn't get to this quicker thanks for cleaning up and merging, hope this PR helps 😄

**Pull request merged**